indico/modules/events/timetable/client/js/EntryPopup.tsx

```diff
@@ -12,79 +12,56 @@ import breakURL from 'indico-url:timetable.tt_break_rest';
 import contributionURL from 'indico-url:timetable.tt_contrib_rest';
 import sessionBlockURL from 'indico-url:timetable.tt_session_block_rest';
 
+import './EntryPopup.module.scss';
 import moment from 'moment';
 import React from 'react';
 import {useSelector, useDispatch} from 'react-redux';
-import {
-  Button,
-  ButtonGroup,
-  Card,
-  Dropdown,
-  DropdownItem,
-  DropdownMenu,
-  Icon,
-  List,
-  Popup,
-  SemanticICONS,
-} from 'semantic-ui-react';
+import {Button, Card, Icon, List, Popup, Label, Header, PopupProps, Image} from 'semantic-ui-react';
 
-import {Translate, Param, Plural, PluralTranslate, Singular} from 'indico/react/i18n';
+import {Translate} from 'indico/react/i18n';
 import './Entry.module.scss';
 import {indicoAxios} from 'indico/utils/axios';
```

Comment:
Move the CSS import to the end, then eslint can auto-sort the imports. Any "with side effects" import unfortunately acts as a sorting boundary...